jenkinsfile

```diff
@@ -60,4 +47,4 @@ pipeline {
       }
     }
   }
-}
+}
```

Comment:
nit: can you add a newline at the end of this file?

Otherwise this change looks fine to me.

Reply:
Done.